test/parallel/test-stream-pipe-await-drain.js

```diff
@@ -27,7 +27,7 @@ writer1.once('chunk-received', () => {
     reader._readableState.awaitDrainWriters.size,
     0,
     'awaitDrain initial value should be 0, actual is ' +
-    reader._readableState.awaitDrainWriters
+    reader._readableState.awaitDrainWriters.size
```

Comment:
this change doesn't seem related?

Reply:
It was actually part of https://github.com/nodejs/node/commit/419686cdfbb3429c512f18bc10c06a8cf5f32cf7, and is required here otherwise the test throws. Ideally I would have submitted a PR to fix it apart from the stream refactor, but I guess I have to learn from past mistakes..